llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -407,6 +407,54 @@ def SYNTACORE_SCR7 : RISCVProcessorModel<"syntacore-scr7",
 
+def TENSTORRENT_ASCALON_D8 : RISCVProcessorModel<"tt-ascalon-d8",
+                                              NoSchedModel,
+                                              [Feature64Bit,
+                                               FeatureStdExtI,
+                                               FeatureStdExtZifencei,
+                                               FeatureStdExtZicsr,
+                                               FeatureStdExtZicntr,
+                                               FeatureStdExtZihpm,
+                                               FeatureStdExtZihintpause,
+                                               FeatureStdExtM,
+                                               FeatureStdExtA,
+                                               FeatureStdExtF,
+                                               FeatureStdExtD,
+                                               FeatureStdExtC,
+                                               FeatureStdExtV,
+                                               FeatureStdExtZvl256b,
+                                               FeatureStdExtZfh,
+                                               FeatureStdExtZvfh,
+                                               FeatureStdExtZba,
+                                               FeatureStdExtZbb,
+                                               FeatureStdExtZbs,
+                                               FeatureStdExtZicbom,
+                                               FeatureStdExtZicbop,
+                                               FeatureStdExtZicboz,
+                                               FeatureStdExtH,
+                                               FeatureStdExtZihintntl,
+                                               FeatureStdExtZfhmin,
+                                               FeatureStdExtZfa,
+                                               FeatureStdExtZkt,
+                                               FeatureStdExtZcb,
+                                               FeatureStdExtZvbb,
+                                               FeatureStdExtZvbc,
+                                               FeatureStdExtZawrs,
+                                               FeatureStdExtZvkng,
+                                               FeatureStdExtZicond,
+                                               FeatureUnalignedScalarMem,
+                                               FeatureUnalignedVectorMem,
+                                               FeatureStdExtSvnapot,
+                                               FeatureStdExtSvpbmt,
+                                               FeatureStdExtSvinval,
+                                               FeatureStdExtZfbfmin,
+                                               FeatureStdExtZvfbfmin,
+                                               FeatureStdExtZvfbfwma],
```

Comment:
Hi, I just did a quick google and it says here that it's rva23 compliant: https://tenstorrent.com/assets/one-pagers/08.21.24_Ascalon.pdf

Should we use RVA23U64Features here? I'm noticing that Zimop is missing in this list though

Reply:
Good point, and we do support Zimop.

Reply:
Zimop addressed, need to double check the rest, there might be more missing.